shopfloor_reception/services/reception.py

```diff
@@ -277,7 +277,8 @@ def _scan_line__find_or_create_line(self, picking, move, qty_done=1):
     def _scan_line__assign_user(self, picking, line, qty_done):
         product = line.product_id
         self._assign_user_to_line(line)
-        line.qty_done += qty_done
+        stock = self._actions_for("stock")
+        stock.mark_move_line_as_picked(line, quantity=qty_done, split=False)
```

Comment:
@rousseldenis with your change in #1023; every time a line is processed, an SQL update on the picking will be done since you always write the `printed` and `user_id` fields on the picking (even if the values are the same)